Automattic/jetpack
# JS Components: Refactor JetpackSearchLogo component

Related to #26481 

#### Changes proposed in this Pull Request:

* Adjust props of JetpackSearchLogo to be consistent with JetpackLogo component

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


N/A

#### Testing instructions:






* Run `cd projects/js-packages/storybook && npm run storybook:dev` and confirm that you can see `JetpackSearchLogo` component in Storybook.
* Ensure that the `JetpackSearchLogo` component works as described.
